### DISCUSSION TITLE
Can't replay join (related to fancy chooser)

### DISCUSSION THREAD

**Discussion by hwine** [saulpw/visidata#934] [Q&A] [answered]
I'm just getting started with visidata, so I'm assuming I'm missing something. I have the "dedupe" (v0.1.0) plugin installed, and want to get a list of folks who have booked multiple appointments. (See [this gist](https://gist.github.com/hwine/cfab8e8ad1b0113e4b0982ca71dfa4f4) for sample file.)

I can do exactly what I want in a manual session, and even successfully save the cmdlog. When I replay the cmdlog, it aborts with an error:
```
S› sheets| no choice made | replay aborted during join-sheets | merge selected sheets with visible columns from all, &  join-sheets         4 sheets
```

My total run is:
```
$ vd t1.tsv        # do everything successfully in manual mode
$ vd -p cmdlog.vd  # reports above error
$ vd --version
saul.pw/VisiData v2.1
```
If I look for an error sheet (via `^E`), none are reported.

I've got no idea what that error means -- any pointers?

Thanks!

**Comment by anjakefala** [comment, marked as answer]
Hey @hwine! 

We have several bugs open for the [fancy chooser not being compatible with replay](https://github.com/saulpw/visidata/issues/914), and unfortunately that is the culprit here. In newer versions of VisiData, I believe it is off by default for that reason.

You can either upgrade or add `options.fancy_chooser = False` to your visidatarc =).

**Comment by anjakefala** [comment]
Btw, thanks for such a detailed bug report! It helped me identify the problem at a quick glance. =)

**Comment by hwine** [comment]
Ah "fancy chooser" -- that's why I couldn't find a bug ;)

Thanks for the quick response -- I had no idea I was so far behind

Setting the option did the trick!
